style.css

```diff
@@ -459,3 +463,148 @@ textarea {
   color: white;
   border: 1px solid #fff;
 }
+
+@media only screen and (min-width: 768px) {
+  #union {
+    display: none;
+  }
+
+  #list-items {
+    display: flex;
+    gap: 3rem;
+    list-style: none;
+  }
+
+  .landing-page {
+    background-color: #fff;
+    border-bottom-left-radius: 10rem;
+    margin-bottom: 6rem;
+
+    /* height: 100vh; */
```

Comment:
- [ ] Kindly remove all commented codes from your documents. It is bad practice to leave commented code in your document.

Reply:
Hello @Kamba56. Thanks for the feedback. I have fixed the spacing now. Can you kindly confirm if it's better?

Reply:
Alright, kindly submit for a re-review.